wolox-training/ap-swift
# Create comment

## Summary ##

**Trello card**: https://trello.com/c/gMir3Vmu/28-vista-para-agregar-comentarios-desde-el-detalle-de-libros-alquilados

Create a new comment for a book

## Test Plan ##

  * Open the app
  * Login with Google
  * The main tab bar view should appear after a successfull login.
  * Press on a rented Book cell
  * The Book Detail  view should appear with a Comment Button
  * Press on Comment Button
  * Comment view should appear

## Screenshots ##

### iPhone 7

![create-comment](https://cloud.githubusercontent.com/assets/25587882/24261034/6c1c2ae4-0fd4-11e7-88be-60adc642ef99.png)

## Screencast ##

![create-comment](https://cloud.githubusercontent.com/assets/25587882/24261048/73b3cf14-0fd4-11e7-8e33-3d61749e6bee.gif)
